### PR TITLE
[#7885] Use old node version for EL7 GH action (main)

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #7885

### CODE REVIEW

File: .github/workflows/build-irods-centos.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 name: build-irods-centos
 on: [push, pull_request]
+env:
+   ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
```
Comment (korydraughn) [inline .github/workflows/build-irods-centos.yml:4]: Let's add a comment which points to the documentation explaining what this is.

**Review by korydraughn — COMMENTED** [review]
Looks good.

**Review by alanking — COMMENTED** [review]
Nice. Will await resolution of open conversation before approving and then we can put this in

File: .github/workflows/build-irods-centos.yml (hunk 1 above)
Comment (SwooshyCueb) [reply .github/workflows/build-irods-centos.yml]: Done

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by SwooshyCueb** [comment]
\#'d

**Review by alanking — APPROVED** [review]

**Pull request merged**